src/crypto/EncryptionSetup.ts

```diff
@@ -365,7 +365,7 @@ class SSSSCryptoCallbacks {
         }
         // if we don't have the key cached yet, ask
         // for it to the general crypto callbacks and cache it
-        if (this.delegateCryptoCallbacks) {
+        if (this.delegateCryptoCallbacks?.getSecretStorageKey) {
```

Comment:
can `delegateCryptoCallbacks` fail? If yes we need to make `this?.delegateCryptoCallbacks?.getSecretStorageKey` instead to check for both cases.

Reply:
As long as `this` is truthy, it won't throw.